test/test_prototype_transforms_consistency.py

```diff
@@ -323,6 +323,9 @@ def __init__(
         ],
         # ElasticTransform needs larger images to avoid the needed internal padding being larger than the actual image
         make_images_kwargs=dict(DEFAULT_MAKE_IMAGES_KWARGS, sizes=[(163, 163), (72, 333), (313, 95)]),
+        # We updated gaussian blur kernel generation with a faster and numerically more stable version
+        # This brings float32 accumulation visible in elastic transform -> we need to relax consistency tolerance
+        closeness_kwargs={"rtol": 1e-1, "atol": 1},
```

Comment:
The `atol` part is problematic here since it basically renders the `float32` test useless: with `atol=1`, every float image within the interval `[0.0, 1.0]` will match regardless of a working kernel or not. I currently don't have a good answer for this problem. See #6750.

I see two options now:

1. Accept that we can't test `float32` for now.
2. Duplicate this whole configuration and set `make_images_kwargs=dict(..., dtypes=[torch.uint8])` and `make_images_kwargs=dict(..., dtypes=[torch.float32])` with the correct tolerances respectively.

Reply:
How about this https://github.com/pytorch/vision/pull/6762/files#diff-bf36f728ca93b501ac1ae0dd786c891c9c389cab57e4843cbd5d3b93a1b95068R311 ?